frontend/compositions/state-input/state-input.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FormEvent, useEffect, useState } from "react"
+import styles from "./state-input.module.css"
+import { FormLevelError } from '../../shared-components/index'
+import { capitalizeFirstChar } from '../../helpers/syntax-helper'
+import { states } from '../../models'
+
+interface USAStateInputProps {
+  isSubmitted: boolean
+}
+export default function USAStateInput({ isSubmitted }: USAStateInputProps) {
+  const {stateSelect, hasError} = styles
+  const [inputId, listId, errorId] = ['stateSelectInput', 'stateSelectList', 'stateSelectError']
+
+  const [inputValue, setInputValue] = useState('')
+  const [isValid, setIsValid] = useState(!isSubmitted)
```

Comment:
Setting isValid to !isSubmitted seems a little counterintuitive to me-- I see that you have checkIsValid below which you're using to set the value of isValid, and you're only updating isValid if isSubmitted. I'm curious if you're anticipating that we'd ever initialize this component with isSubmitted=true? 
In general can you explain why you  did it this way?

Reply:
Hmm, I can't quite remember honestly - I remember it used to be set to state `checkIsValid('')` but I ran into some sort of issue with that. That issue could be outdated by now though honestly so I'll try again!

Reply:
To me it makes more senses to initialize isValid to "true", unless there's a particular reason not to!

Reply:
To clarify for other folks, addressed this with emma to use `checkIsValid()` calls onInit!